src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
+    int ballNumber;
```

Comment:
```suggestion
    private String color;
    private int number;
```
Renamed ballColor -> color and ballNumber -> ball because we are in Ball class and know that color and number are properties of ball. The access modifiers of the color and number fields must been changed to private to ensure that access to them is controlled through getters and setters.